en/docs/concepts/revoked-tokens.md

```diff
@@ -0,0 +1,25 @@
+# Revoked Tokens
+
+There are two usecase where API Microgateway needs to know when a token is revoked by the Security Token Service (STS).
```

Comment:
The API Microgateway is notified when a token is revoked by the Security Token Service (STS) as described in the following two instances.